WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -97,4 +105,26 @@ public void onJSONException(JSONException exception) {
             }
         });
     }
+
+    @Override
+    public void onPersonSelected(Person person) {
+        FragmentManager fragmentManager = getFragmentManager();
+        PersonDetailFragment personDetailFragment = (PersonDetailFragment) fragmentManager.findFragmentByTag(KEY_PERSON_DETAIL_FRAGMENT);
+
+        long personID = person.getPersonID();
+        int localTableBlogID = person.getLocalTableBlogId();
+        if (personDetailFragment == null) {
+            personDetailFragment = PersonDetailFragment.newInstance(personID, localTableBlogID);
+        } else {
+            personDetailFragment.setPersonDetails(personID, localTableBlogID);
+        }
+        if (!personDetailFragment.isAdded()) {
+            FragmentTransaction fragmentTransaction = getFragmentManager().beginTransaction();
+            fragmentTransaction.setCustomAnimations(R.animator.fragment_slide_in_from_right, R.animator.fragment_slide_out_to_left,
```

Comment:
@mattmiklic The question was either to use a consistent slide in-out animation for the people management pages or to just let the OS decide on the animation (which would be different) because of technical issues. I think we fixed most of the issues there, so I guess you can ignore the ping :)